### DISCUSSION TITLE
General discussion for adsbmilanalytics.com

### DISCUSSION THREAD

**Discussion by EMcNugget** [EMcNugget/adsb_mil_data#5] [General]
One stop shop for anything relating to this repo.

**Comment by krgnl** [comment]


**Reply by EMcNugget** [reply]
If you wish to have a serious conversation I’m not willing to do it via that email address as that’s very suspicious. Or you could just put your ideas here.

**Reply by EMcNugget** [reply]
I see, I’ll go ahead and shoot you an email.
